homeassistant/components/cloud/alexa_config.py

```diff
@@ -146,7 +150,7 @@ def __init__(
         self._cloud_user = cloud_user
         self._prefs = prefs
         self._cloud = cloud
-        self._token = None
+        self._token: str | None = None
```

Comment:
This was an existing issue that mypy started to complain about now as the new handler have proper types.